test/integration/acceptance/custom/basic/basic.go

```diff
@@ -120,10 +120,12 @@ func (r *BasicTestRunner) Delete(ctx context.Context, t *testing.T) {
 	defer cn()
 	pub1Cluster, _ := r.GetPublicContext(1)
 	prv1Cluster, _ := r.GetPrivateContext(1)
-	assert.Assert(t, pub1Cluster.VanClient.SiteConfigRemove(ctx))
-	assert.Assert(t, pub1Cluster.VanClient.RouterRemove(ctx))
-	assert.Assert(t, prv1Cluster.VanClient.SiteConfigRemove(ctx))
-	assert.Assert(t, prv1Cluster.VanClient.RouterRemove(ctx))
+	if err := pub1Cluster.VanClient.SiteConfigRemove(ctx); err != nil {
+		_ = pub1Cluster.VanClient.RouterRemove(ctx)
+	}
+	if err := prv1Cluster.VanClient.SiteConfigRemove(ctx); err != nil {
+		_ = prv1Cluster.VanClient.RouterRemove(ctx)
+	}
 	waitNoPods := func(componentSelector string, cluster *base.ClusterContext) error {
```

Comment:
I don't see the reason for the new `if`, if the `Assert`s are gone.  This way, `pub1Cluster.VanClient.RouterRemove` will never be tried if `pub1Cluster.VanClient.SiteConfigRemove` fails.

Removing the `if`, in the case that `pub1Cluster.VanClient.SiteConfigRemove` fails, would you get a second chance of getting rid of the skupper installation with `pub1Cluster.VanClient.RouterRemove`?  Or would that not be sufficient, and that's the reason for the `if`?

It's good that the `Assert` statements are gone, so that an otherwise working test does not fail on a cleanup task.  However, I think we need some logs in case of errors there.  Otherwise, it may be difficult to understand why the next test fails when the removal does not succeed.

Reply:
Actually if SiteConfigRemove fails, the RouterRemove will be performed, and only if it fails.
If the controller is running properly, just removing the site-config should be enough. That's 
exactly what skupper delete does in fact.

Reply:
Ok, I will add it.